### ISSUE TITLE
Work with oc 10?

### ISSUE THREAD

**Issue by magicWF** [buzz/ics2owncloud.py#22]
Hello,

i tried the script with oc 10, but it does not work. No imported items, no errors!?

Regards
Maik

**Comment by josefw** [comment]
Same problem here. Any news?

**Comment by buzz** [comment]
Sorry, this was just meant as a stop-gap solution until OC supports webcal natively. OC does now so I'm not using this anymore personally. Please debug yourself.

**Comment by tboerner** [comment]
1. Python:
The script uses Python 3 syntax, but the virtualenv provides Python 2.7. 
Either adapt it to 2.7: 
replace #!/usr/bin/env python3 with #!/usr/bin/env python2
add
from __future__ import print_function
and replace configparser with ConfigParser.
Or, if you have Python 3 installed, run the script directly (not with the shell script).
Make sure you have python3-icalendar installed.

2. Owncloud:
The base_url has changed. It is now:
CALDAVURL = '%sremote.php/dav/calendars/%s/%s/'

**Comment by buzz** [comment]
> 1. Python:
>    The script uses Python 3 syntax, but the virtualenv provides Python 2.7.
>    Either adapt it to 2.7:
>    replace #!/usr/bin/env python3 with #!/usr/bin/env python2
>    add
>    from **future** import print_function
>    and replace configparser with ConfigParser.
>    Or, if you have Python 3 installed, run the script directly (not with the shell script).
>    Make sure you have python3-icalendar installed.
> 2. Owncloud:
>    The base_url has changed. It is now:
>    CALDAVURL = '%sremote.php/dav/calendars/%s/%s/'

If you can create a PR I'm happy to pull.

**Comment by tboerner** [comment]
Yes I can - having much to do these days, it will probably happen in January.

Am 11. Dezember 2019 21:26:52 MEZ schrieb buzz <notifications@github.com>:
>> 1. Python:
>>    The script uses Python 3 syntax, but the virtualenv provides
>Python 2.7.
>>    Either adapt it to 2.7:
>>    replace #!/usr/bin/env python3 with #!/usr/bin/env python2
>>    add
>>    from **future** import print_function
>>    and replace configparser with ConfigParser.
>>    Or, if you have Python 3 installed, run the script directly (not
>with the shell script).
>>    Make sure you have python3-icalendar installed.
>> 2. Owncloud:
>>    The base_url has changed. It is now:
>>    CALDAVURL = '%sremote.php/dav/calendars/%s/%s/'
>
>If you can create a PR I'm happy to pull.
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/buzz/ics2owncloud.py/issues/22#issuecomment-564718193

-- 
Thomas Börner 
Werderstr. 8 
35037 Marburg

Tel +49 (6421) 682104  
Fax +49 (6421) 682114  
mailto:thomas@boerner.click 
GPG: https://crypt.boerner.click (Fingerabdruck: 59EC 235C 949C AF4D 387D 40CF 29F7 2265 3197 8E56)
